express/src/controllers/register.ts

```diff
@@ -223,3 +261,27 @@ export const accountExists: RequestHandler = (req, res) => {
         }
     });
 };
+
+export const resumeUserJourneyAfterPassword: RequestHandler = async (req, res) => {
+    const s4: SelfServiceServicesService = req.app.get("backing-service");
+
+    console.log("In resumeUserJourney");
+    const userName = nonNull(req.session.emailAddress);
+    const userPassword = nonNull(req.body["password"]);
+    console.log("User Name => " + userName);
+    console.log("User Password => " + userPassword);
+
+    const response = await s4.submitUsernamePassword(userName, userPassword);
+    req.session.cognitoSession = response.Session;
+    req.session.authenticationResult = response.AuthenticationResult;
+    //req.session.mobileNumber = await s4.getMobilePhoneNumber(userName);
+
+    // Keith Hart has requested even if User has already entered their phone number we 'resume' from re-entering their Phone number again.
```

Comment:
I think these explanations are better put on the ticket and not in the code. Commented code should be removed.